packages/nx/src/command-line/connect/connect-to-nx-cloud.ts

```diff
@@ -8,16 +8,29 @@ import {
   isNxCloudUsed,
 } from '../../utils/nx-cloud-utils';
 import { runNxSync } from '../../utils/child-process';
+import { NxJsonConfiguration } from '../../config/nx-json';
+
+export function onlyDefaultRunnerIsUsed(nxJson: NxJsonConfiguration) {
+  if (!nxJson.tasksRunnerOptions) {
+    if (!nxJson.nxCloudAccessToken) return false;
+    return true;
+  }
```

Comment:
This seems backwards..

They are using the default task runner if they don't have `taskRunnerOptions` AND they don't have the `nxCloudAccessToken`.